docs/source/notebooks/Embedding_nets.ipynb

```diff
@@ -0,0 +1,420 @@
+    "        return self.net(x)\n",
+    "\n",
+    "embedding = FullyCOnnectedNetwork(np.sum([data.wavelength.shape[0] for data in dataset]))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can find some further information on defining embedding networks in the SBI documentation at https://www.mackelab.org/sbi/tutorial/05_embedding_net/.\n",
+    "\n",
+    "However, SBI also packages a small selection of pre-defined networks that you can use, and we can specify the same network by just importing one of those and passing it the same setup, for example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sbi.neural_nets.embedding_nets import FCEmbedding\n",
+    "\n",
+    "embedding = FCEmbedding(np.sum([data.wavelength.shape[0] for data in dataset]), output_dim=20, num_layers=3, num_hidden_units=100)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is already a lot easier, at least as long as one of the architectures provided by SBI (Fully Connected and Convolutional Neural Networks) is suitable for your problem.\n",
+    "\n",
+    "But to try and make life even easier, Ampere can construct an embedding for you, using the architectures packaged by SBI!\n",
+    "You can simply accept the default hyperparameters for these networks by passing either `embedding_net='CNN'` or `embedding_net='FC'` when instantiating the inference object, or you can pass Ampere a dictionaary defining the values of the hyperparameters that you want to change along with an indication of the type of network you want to use:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "embedding = {'type': 'FC', 'num_hiddens': 100, 'n_layers': 3, \"output_dim\": 20}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And now things proceed in a very similar fashion to a normal NPE run! \n",
+    "The only difference is that our dictionary telling Ampere how to define the embedding is passed along with the model and dataset when instantiating our inference object. \n",
+    "The dict could also just be replaced with a user-defined network or the strings indicated above for default hyperparameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer = SBI_SNPE(model=model, data=dataset, embedding_net=embedding)\n",
+    "\n",
+    "# Then we tell it to explore the parameter space\n",
+    "optimizer.optimise(nsamples=10000, nsamples_post=10000, n_rounds=1\n",
+    "                    )\n",
+    "\n",
+    "# now we call the postprocessing to produce some figures\n",
+    "optimizer.postProcess()"
```

Comment:
Not directly related to the optimizer but to the plotting routines. In order for the user to determine whether the fit is reasonable, the final spectrum/SED plot needs to be scaled to the range of data values rather than model values. The current scaling defeats the purpose of the example being demonstrated in this notebook. See resulting plot attached here.
![Screenshot 2023-07-21 at 9 45 05](https://github.com/ICSM/ampere/assets/15619959/a2efc72d-b2e3-44a1-a850-f77634d06939)

Reply:
This should be handled in f0133b43e9cb17ad756f0f2aa461fb821ffc4a4f and 5c50da174259a8df70f5fd879ab8095a110e503c on main.